taipy/core/submission/submission.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         self,
         entity_id: str,
         entity_type: str,
+        entity_config_id: Optional[str] = None,
```

Comment:
Would it make more sense to use a Properties object to store this kind of additional/optional information?
-> hmm I don't think so as it's a system attribute instead of an additional/optional information. So putting it in the `_Properties` will put a different semantic meaning to the field.

`Properties` might well become a field of the `_Entity` type
-> That's a valid point. But we don't have `_Properties` for all entity classes (`_Version`, `Submission`, `Job`)

Reply:
> So putting it in the _Properties will put a different semantic meaning to the field.

That was exactly my point. This field is optional and does not make sense in every cases. It could perfectly match the `Properties` purpose.